patches/nginx-1.11.2-privilege_process.patch

```diff
@@ -0,0 +1,208 @@
+diff --git a/src/core/nginx.c b/src/core/nginx.c
```

Comment:
Also, I think the patch file name should be `nginx-1.11.2-privileged_agent_process.patch` instead.

Never mind, I'll rename it locally myself.